TASVideos.WikiEngine/NodeImplementations/Module.cs

```diff
@@ -65,6 +65,33 @@ public async Task WriteTextAsync(TextWriter writer, WriterContext ctx)
 		}
 	}
 
+	public async Task WriteMetaDescriptionAsync(StringBuilder sb, WriterContext ctx)
+	{
+		if (sb.Length == SiteGlobalConstants.MetaDescriptionLength)
+		{
+			return;
+		}
+
+		if (ModuleNames.IsModule(Name))
+		{
+			StringWriter sw = new();
```

Comment:
Could this pass through the parent `TextWriter` instead of making a new one?

Reply:
What parent `TextWriter`?

Reply:
This is assuming the other comment is acted on; then there would be a parent `TextWriter` passed in here.

Reply:
Resolving as we'll keep the stringbuilder as discussed elsewhere.